elasticdl/cc/common/type.cc

```diff
@@ -0,0 +1 @@
+#include "elasticdl/cc/common/type.h"
```

Comment:
Why need `type.cc` which contains only one line inlude ?

Reply:
Fix it.